examples/with-i18next/pages/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react'
+import { I18nextProvider } from 'react-i18next'
+import startI18n from '../tools/startI18n'
+import { getTranslation } from '../tools/translationHelpers'
+import Title from '../components/Title'
+
+export default class Homepage extends Component {
+  static async getInitialProps ({ req }) {
+    const isServer = !!req
+    const translations = await getTranslation('pt', 'common', 'http://localhost:3000/static/locales/')
```

Comment:
Do we need to express the full url? What happens if we deploy to a non localhost env? for example we suggest you can use zeit to deploy